### ISSUE TITLE
Uncaught error when trying to use inheritance without specifying base constructor arguments

### ISSUE THREAD

**Issue by filippsen** [SuperblocksHQ/superblocks-lab#101]
## Environment/Browser
Version 1.0.0

## Description
>"Trying to use inheritance in my Smart Contract but I can’t get it to work"

The error happens when the base contract specifies constructor arguments and the inherited contract doesn't specify anything.

## Steps to reproduce
1. _Create New_ project from _Hello World_ template
2. Add _New Contract_ to the project named: `Something`
3. Edit `Something.sol` contents to match the following code:
```
pragma solidity ^0.4.17;

import "./HelloWorld.sol";

contract Something is HelloWorld {
}
```
4. Click _Compile_

## Expected result
> Success in compilation

## Actual result
> index-v0.4.21-1.html:1 Uncaught SyntaxError: Unexpected end of JSON input
   at JSON.parse (<anonymous>)
   at eval (webpack-internal:///./components/projecteditor/compiler.js:206)
   at Solc._response (webpack-internal:///./components/solc/index.js:39)
   at setTimeout (solc-iframe-v0.4.21-1.js:10264)

**Comment by filippsen** [comment]
It looks like the compiler queue understand the contract object is good, when it isn't. The contract object "exists", but it is empty, so the simple check `if(contractObj)` passes.
At that point, the contract object is in a bad state and breaks. The code is valid.

Doing the other way around `contract HelloWorld is Something` works.

**Comment by filippsen** [comment]
According to the specification, while the code is valid, the correct code would involve setting the base constructor arguments when declaring contract inheritance.

```
pragma solidity ^0.4.17;

import "HelloWorld.sol";

contract Something is HelloWorld("") {
}
```

### What should be fixed ? 
The compiler, as it is, fails and hangs without throwing any visible error. If the code is insufficient, an error should be thrown.

### Reference
https://solidity.readthedocs.io/en/develop/contracts.html#arguments-for-base-constructors

**Comment by filippsen** [comment]
Solved in #262
